frontend/components/navigationButtons.jsx

```diff
@@ -8,8 +8,8 @@ const NavWrapper = styled.nav`
 `
 
 const NavigationButtons = ({ currentQuestionId, surveyLength }) => {
-  const prevQUrl = `/survey/questions/${currentQuestionId - 1}`
-  const nextQUrl = `/survey/questions/${currentQuestionId + 1}`
+  const prevQUrl = `/survey/questions/question/?id=${currentQuestionId - 1}`
+  const nextQUrl = `/survey/questions/question/?id=${currentQuestionId + 1}`
```

Comment:
URLs could be /survey/question/?id=... not /survey/questions/question/... ?

Reply:
BTW: is request changes ok for minor requests like these or more so for bigger issues only?

Reply:
I think "/survey/questions/question/?id=x" is logical since there is "/survey/questions/summary/" page as well.

Reply:
> I think "/survey/questions/question/?id=x" is logical since there is "/survey/questions/summary/" page as well.

This was my reasoning